README.md

```diff
@@ -79,6 +80,16 @@ Response (in Body):
 }
 ```
 
+#### Envelope Statuses
+
+The following are the possible Envelope Statuses that occur for an Envelope.
+
+| Status  | Description  | 
+| --------|---------|
+| Open  |  Envelope is created and open to upload files.  |
+| Closed |  The result of the envelope "Sealed". Any attempts to upload a file will result with a 403 (Locked) |
```

Comment:
OPEN -> descrption is fine
SEALED -> routing request was made but not files are AVAILABLE yet. Any attempts to upload a file will result with a 403 (Locked)
CLOSED -> routing request was made and all files are AVAILABLE. Any attempts to upload a file will result with a 403 (Locked)
DELETED -> description is fine